README.md

```diff
@@ -48,6 +48,11 @@ In .env file add variables:
 
 .env file is in .gitignore.
 
+Make sure that dotenv module is installed using:
+```bash
+pip3 install python-dotenv
```

Comment:
We have this in requirements/dev.txt, right? When we run `pip install -r requirements/dev.txt` python-dotenv==0.21.0 will be installed

Reply:
correct, removed that from readme